backend/domain/use_cases/admin_usecases.py

```diff
@@ -0,0 +1,14 @@
+from domain.entities.admin import Admin
+from domain.repositories.admin_repository import AdminDbRepository
+from domain.use_cases.work_with_password import WorkWithPassword
+from flask import jsonify
+
+
+class AdminRegister(Admin):
```

Comment:
It's a good idea to create a class (or a module) that is responsible for communication with a database and providing a high level abstraction on top of SQL queries. However using inheritance here is not justified. (we discussed the topic today, see the links in the chat.)

There is already a module `work_with_db.py ` that has some functions to work with a db.

I would recommend to take a functionality from it, take the function admin_register and put it into a new module. That module would handle admin records (one responsibility).
Make an interface of the module uniform. For example, it's either just 2 standalone functions, or a class with two methods.